### ISSUE TITLE
Supported and accepted rels

### ISSUE THREAD

**Issue by prologic** [webfinger/webfinger.net#31]
Could someone please help me understand what the accepted `rel`(s) are? Currently the neither the spec nor [webfinger.net](https://webfinger.net) _really_ make this clear. Are they:

- `avatar`
- `profile-page`

Or:

- `https://webfinger.net/avatar`
- `https://webfinger.net/profile-page`

Thanks!

**Comment by pfefferle** [comment]
neither one of them, they are:

* `http://webfinger.net/rel/avatar`
* `http://webfinger.net/rel/profile-page`

**Comment by pfefferle** [comment]
This is mentioned on the rel page itself... but maybe it is not dominant enough, because of the very decent highlighting of the URL...

**Comment by prologic** [comment]
> This is mentioned on the rel page itself... but maybe it is not dominant enough, because of the very decent highlighting of the URL...

Actually you are right, however, the reason I posted this and asked is because the spec itself is confusing. It makes it appear as though "rel"(s) are themselves URI(s) of some kind and not just a commonly accepted name 🤦‍♂️ -- Its as-if the folks that wrote the spec aren't the same as the ones that built the webfinger.net site and this Github project 🤣

**Comment by prologic** [comment]
Anyway I've used `avatar` and `profile-page` as per the landing page 👌

**Comment by pfefferle** [comment]
But the spec says:

> The value of the "rel" member is a string that is either a URI or a registered relation type [8] (see RFC 5988 [4]).  The value of the "rel" member MUST contain exactly one URI or registered relation type.  The URI or registered relation type identifies the type of the link relation.

And that's what the https://webfinger.net/rel pages says, there are two common links defined using the WebFinger.net "namespace" but you can also use one of the registered rel-values.

**Comment by pfefferle** [comment]
But I agree, that the two values you mentioned could be misinterpreted and perhaps we should change them to the full URI!

**Comment by willnorris** [comment]
Yeah, you're totally right.  Updated those to the full URIs.

**Comment by willnorris** [comment]
yes, use the full URIs.  See for example, https://webfinger.net/lookup/?resource=will@willnorris.com

**Comment by prologic** [comment]
Btw since I'm also building a combined client+server library at https://git.mills.io/prologic/webfinger -- Are there any other rels that are commonly accepted or supported by WebFinger servers/clients we should be listing on the "Rels" page? 🤔 Should `Self` be there for example? 🤔

**Comment by willnorris** [comment]
I've just created #32 to cleanup the rel page a little bit.  Originally, this page was only created to document the two rel values that were created in the webfinger.net namespace.  It also now links to the official IANA registry that has all registered rel values: https://www.iana.org/assignments/link-relations/

We could theoretically list other commonly used rel values, but since webfinger is really intended to be a generic protocol for discovering links and metadata, it's hard to say what would be common.  Right now, fediverse related use cases are certainly dominating webfinger usage, but it was never intended to be limited to that.

**Comment by prologic** [comment]
Appreciate the quick responses too guys ! 🙇‍♂️ I'm all good now 🤞
